internal/workload/network/netfilter.go

```diff
@@ -66,13 +66,17 @@ func (nft *netfilter) DeleteTable(table string) error {
 }
 
 func (nft *netfilter) AddChain(table, chain string) error {
-	args := []string{"add", "chain", family, table, chain, "{ type filter hook input priority 0 ; }"}
+	// Fixes https://github.com/project-flotta/flotta-device-worker/issues/216
+	chainName := fmt.Sprintf("wl-%s", chain)
```

Comment:
Pls extract `fmt.Sprintf("wl-%s", chain)` to a method.
It is repeated twice and should be defined only once to avoid future issues.

Reply:
Will do.

Reply:
Done